legend-engine-shared-core/src/main/java/org/finos/legend/engine/shared/core/identity/Credential.java

```diff
@@ -23,4 +23,10 @@ default boolean isValid()
     {
         return true;
     }
+
+    default <T> T accept(CredentialVisitor<T> consumer)

```

Comment:
variable name. Also can we have default implementation as 
`visitor.visit(this)`

Reply:
i changed it to abstract function.